package.json

```diff
@@ -1,10 +1,11 @@
 {
   "author": "The Daily Beast",
   "dependencies": {
-    "react": "16.2.0",
-    "react-dom": "16.2.0",
     "uuid": "^3.2.1"
   },
+  "peerDependencies": {
+    "react": "^16.2.0"
+  },
```

Comment:
Hmm ok so this package is available to the public. I am sure not everyone will use 16.2. Should this just use any version greater than 16.2 without issue?

Also, if we update the react version in our code, will mobiledoc-react-renderer automatically start using the new version? Pardon my ignorance.

Reply:
I looked into it, and it sounds like it should do what I described. https://nodejs.org/en/blog/npm/peer-dependencies/

I guess it is kind of us to make sure that there aren't problems as we increase the react version.

Reply:
no worries. so the way it was setup before, it required a very specific version of react. so if the consuming project was using react@16.3, they would also be bundling react@16.2.0 into their project(yes, two copies of react) because that is the version specified in this project. the caret in front of the version number tells npm that this project needs any version of react at 16.2 or above but lower than v17. i'm pretty sure we can open this to any version of 16 (^16.0.0) but just to be safe, i limited it to 16.2 and above as it previously set

Reply:
Yeah that confirms what I thought the docs were saying. In that case it seems well suited to anyone using this package outside the daily beast.